sdk/metric/instrument.go

```diff
@@ -194,8 +194,12 @@ func (i *instrumentImpl[N]) aggregate(ctx context.Context, val N, attrs []attrib
 	if err := ctx.Err(); err != nil {
 		return
 	}
+	// Do not use single attribute.Sortable and attribute.NewSetWithSortable, this
+	// method needs to be concurrent safe. Let the attribute package pool
+	// handle allocations of the Sortable.
```

Comment:
Not a native speaker but maybe this would be more clear (I have doubts. but I had to read this sentence a few times)

```suggestion
	// method needs to be concurrent safe. Let the sync.Pool
	// in the attribute package handle allocations of the Sortable.
```

PS. The same comment applies to the second occurrence of this comment.